webpack.config.prod.js

```diff
@@ -1,26 +1,33 @@
 var webpack = require('webpack');
 var config = require('./webpack.config');
 
-config.plugins.push(new webpack.optimize.UglifyJsPlugin({
-    minimize: true,
-    output: {
-        comments: false,
-        screw_ie8: true
-    },
-    mangle: {
-        screw_ie8: true
-    },
-    compress: {
-        warnings: false,
-        screw_ie8: true
-    }
-}));
+config.plugins.push(
+    new webpack.DefinePlugin({
```

Comment:
Setting this environment variable drop some Vue.js code (warnings and some debug helpers)  in production.